odiglet/pkg/instrumentation/fs/remove.go

```diff
@@ -4,43 +4,59 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"strings"
 
 	"github.com/odigos-io/odigos/odiglet/pkg/log"
 )
 
-func removeFilesInDir(hostDir string) error {
+func removeFilesInDir(hostDir string, filesToKeep map[string]struct{}) error {
 	shouldRecreateCFiles := ShouldRecreateAllCFiles()
-	log.Logger.V(0).Info(fmt.Sprintf("Removing files in directory: %s, shouldRecreateCFiles: %s", hostDir, fmt.Sprintf("%t", shouldRecreateCFiles)))
+	log.Logger.V(0).Info(fmt.Sprintf("Removing files in directory: %s, shouldRecreateCFiles: %t", hostDir, shouldRecreateCFiles))
```

Comment:
nit: probably better to use structured logging (arguments as key-value pairs instead of one formatted string)